streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/core/graph/executiongraph/ExecutionJobVertex.java

```diff
@@ -22,42 +25,54 @@
 public class ExecutionJobVertex {
 
   /**
-   * Unique id for execution job vertex.
+   * Unique id of operator(use {@link JobVertex}'s id).
    */
-  private final int jobVertexId;
+  private final int operatorId;
```

Comment:
Can you elaborate more why we need to rename `jobVertexId ` to `operatorId `?

Reply:
My original intention was to make this field easier to understand.
But it doesn't really mean much after some considering.

Reply:
Revert it.